api/omnibox/input-suggestions/background.js

```diff
@@ -0,0 +1,23 @@
+// Copyright 2021 Google LLC
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file or at
+// https://developers.google.com/open-source/licenses/bsd
+
+// This event is fired each time the user updates the text in the omnibox,
+// as long as the extension's keyword mode is still active.
+chrome.omnibox.onInputChanged.addListener((text, suggest) => {
+  console.log('inputChanged: ' + text);
+  suggest([
+    {content: text + " one", description: "the first one"},
+    {content: text + " number two", description: "the second entry"}
+  ]);
+});
+
+// This event is fired with the user accepts the input in the omnibox.
+chrome.omnibox.onInputEntered.addListener((text) => {
+  console.log('inputEntered: ' + text);
+  chrome.tabs.create({
+    url: `data:text/html;charset=utf-8,You entered "${text}"`
```

Comment:
Sorry to make you expand this, but can you make this a real page with a query-string?

That page would be included in this extension and have a tiny note about "You can open a new tab in response to an omnibox search, but it's not the only possibility". I think it might confuse our users who think the only thing you can do is open a page.